__docs__/wonder-blocks-tokens/semantic-color.stories.tsx

```diff
@@ -87,8 +87,6 @@ export const SemanticColors = () => (
                 cell: (row) => (
                     <View
                         style={{
-                            background:
-                                semanticColor.core.background.neutral.subtle,
```

Comment:
Removed the additional background color for the color docs! Sharing from the related [Chromatic comment](https://www.chromatic.com/test?appId=5e1bf4b385e3fb0020b7073c&id=68a8c406fcaa973a83bdd04b):

> I removed the gray box around the color box!
>
> I was checking the instances of semanticColor.background.neutral.subtle  since the value will be updated from white to light gray in TB. 
>
> I find the colors easier to see without the gray background so I opted to remove the background box color altogether!